index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main class="container">
+        <header class="header-background">
+            <div class="logo-placeholder">
+                <a href="/index.html">Mark</a>
+            </div>
+            <nav class="icon-menu">
+                    <div class="menu-icon">
+                        <img src="./images/menu.svg" alt="menu icon" class="nav-icon">
+                    </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1 class="username">I'm Mark, Glad to see you!</h1>
+            </div>
+            <div class="description">
+                <p class="user-description">
+                    I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
Please use the font family "Poppins" as per requirement for the paragraph. 👍🏻 

[Font, you used]
<img width="289" alt="Screen Shot 2022-04-25 at 4 03 40 PM" src="https://user-images.githubusercontent.com/83048208/165076856-61333bdc-ceab-4116-b803-457e3b8d4d6e.png">

[Required Font]
<img width="387" alt="Screen Shot 2022-04-25 at 4 04 33 PM" src="https://user-images.githubusercontent.com/83048208/165076975-dfbf9351-b9b5-4e69-8180-7b41a258cb1e.png">